assets/src/edit-story/components/colorPicker/useKeyDeleteStop.js

```diff
@@ -27,7 +27,7 @@ function useKeyDeleteStop(ref, onDelete) {
       ref.current.focus();
       onDelete();
     },
-    []
+    [onDelete, ref]
```

Comment:
Not sure, I'd have to check how the rule works under the hood.

Reply:
We can skip this for now, but we do have a lot of those around - would be awesome if we could however set the linter up to assume, that anything named `/[Rr]ef$/` and only accessed via `.current` is a stable ref and never needs to be in dependencies. It might be wishful thinking though.

Reply:
+1 to not have `refs` as deps. but in this case it arrives from outside so there's a limited info that this is indeed a ref.